src/core/short_range_loop.hpp

```diff
@@ -38,14 +38,17 @@ struct True {
 template <class BondKernel, class PairKernel,
           class VerletCriterion = detail::True>
 void short_range_loop(BondKernel bond_kernel, PairKernel pair_kernel,
-                      double distance_cutoff,
+                      double pair_cutoff, double bond_cutoff,
                       const VerletCriterion &verlet_criterion = {}) {
   ESPRESSO_PROFILER_CXX_MARK_FUNCTION;
 
   assert(cell_structure.get_resort_particles() == Cells::RESORT_NONE);
 
-  cell_structure.bond_loop(bond_kernel);
-  if (distance_cutoff > 0.)
+  if (bond_cutoff >= 0.) {
```

Comment:
Why use `>=` here to include zero, now that bonded IAs without range return `0.` as the cutoff instead of `-1.`?
Could it be a mistake, i.e. bonded IAs w/o range should now return `BONDED_INACTIVE_CUTOFF` instead of `0.`?

Reply:
including the zero does not make sense to me